src/VisualStudio/Core/Def/EditAndContinue/EditAndContinueFeedbackDiagnosticFileProvider.cs

```diff
@@ -65,6 +65,12 @@ internal sealed class EditAndContinueFeedbackDiagnosticFileProvider : IFeedbackD
         var vsFeedbackTempDir = Path.Combine(_tempDir, VSFeedbackSemaphoreDir);
         _vsFeedbackSemaphoreFullPath = Path.Combine(vsFeedbackTempDir, VSFeedbackSemaphoreFileName);
 
+        // Directory may not exist in scenarios such as Razor integration tests
+        if (!Directory.Exists(vsFeedbackTempDir))
```

Comment:
File watcher throws when the directory does not exist? What is the exact exception trace?
That'd be unfortunate since the intent was to detect when the file in that directory is created. The directory indeed might not exist at this time because VS Feedback didn't created it yet.

Reply:
I see. ok.